hypershift-operator/controllers/nodepool/config.go

```diff
@@ -353,6 +353,7 @@ func globalConfigString(hcluster *hyperv1.HostedCluster) (string, error) {
 	// This PR drops the field when empty from the mashaled string to keep backward compatibility.
 	// Implementing this at the marshal operation level might result in undesired impact as we might potentially modify other fields and ordering is not deterministic
 	str := globalConfigBytes.String()
-	str = strings.ReplaceAll(str, ",\"imageStreamImportMode\":\"\"", "")
+	// str = strings.ReplaceAll(str, ",\"imageStreamImportMode\":\"\"", "")
```

Comment:
please let's remove commented out code

Reply:
oops, missed that one! good catch! glad I didn't miss the "this is a dirty hack" debug logs too 😄